src/window.rs

```diff
@@ -682,6 +682,26 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused. How requesting for user attention manifests is platform dependent,
+    /// see `UserAttentionType` for details.
+    ///
+    /// Providing `None` will unset the request for user attention.
+    ///
+    /// Unsetting the request for user attention might not be done automatically by the WM
+    /// when the window receives input.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web / Wayland:** Unsupported.
+    /// - **macOS:** `None` has no effect.
+    /// - **X11:** Automatic unsetting the request for user attention when the window  
+    /// receives focus might not be handled by some WMs.
```

Comment:
Might be easier to just state this as a fact. Like "Attention requests must be manually cleared". That way, people are automatically encouraged to do it right on X11.